src/notes-navigation-item.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="notes-nativation-item.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.5.2/css/all.min.css"
+      integrity="sha512-SnH5WK+bZxgPHs44uWIX+LLJAJ9/2PkPKZ5QiAj6Ta86w+fsb2TkcmfRyVX3pBnMFcV7oQPJkl9QevSCWr3W6A=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <title>Notes app</title>
+  </head>
+  <body>
+    <nav id="navbar" class="navigation__bar">
```

Comment:
Use correct BEM css notation for naming. Please see example below. Also modify in the css file.
.navigation__list {}
.navigation__list__item {}
 .navigation__list__item--active {}
 <li class="navigation__list__item navigation__list__item--active">